resources/dw/schema_cluster.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2024 Cloudera. All Rights Reserved.
```

Comment:
to avoid accommodating the azureClusterSchema here you might rename this file to `schema_cluster_aws.go`.